src/masternodeconfig.cpp

```diff
@@ -16,41 +16,42 @@ bool CMasternodeConfig::read(boost::filesystem::path path) {
     if (!streamConfig.good()) {
         return true; // No masternode.conf file is OK
     }
-
+ 
     for(std::string line; std::getline(streamConfig, line); )
     {
         if(line.empty()) {
             continue;
         }
         std::istringstream iss(line);
-        std::string alias, ip, privKey, txHash, outputIndex;
-        iss.str(line);
-        iss.clear();
-        if (!(iss >> alias >> ip >> privKey >> txHash >> outputIndex)) {
-            LogPrintf("Could not parse masternode.conf line: %s\n", line.c_str());
-            streamConfig.close();
-            return false;
-        }
-
-        if(Params().NetworkID() == CChainParams::MAIN){
```

Comment:
this is the part I was speaking about, which checks the port. one more to check that it is on an IPV4 ;)